src/scrapers/leumi.ts

```diff
@@ -1,19 +1,22 @@
-import moment, { Moment } from 'moment';
-import { Page } from 'puppeteer';
-import { BaseScraperWithBrowser, LoginResults, LoginOptions } from './base-scraper-with-browser';
+// eslint-disable-next-line import/named
+import moment, { type Moment } from 'moment';
```

Comment:
De a search to remove all the `eslint-disable-next-line import/named`

```suggestion
import moment, { type Moment } from 'moment';
```